### PR TITLE
updated logo component using next.js

### LINKED ISSUE #13: Logo component

### PR DESCRIPTION
**Pull request by Asel-K**
I finally could create the logo component using next,js.  I've also added all types of the logo ("W' with and without name of the app in darker and lighter colors) into the "public/images" file. Please take a look and if this works I will delete my previous pull request on logo component.

### CODE REVIEW

File: components/layout/Logo.js [hunk 1]
```diff
@@ -1,3 +1,16 @@
-const Logo = () => <div>Site logo here</div>;
+import Image from 'next/image'
 
-export default Logo;
+const Logo = () => {
+    return (
+        <>
+            <Image 
+            src="/../public/images/Whatodogrey.svg"
```
Comment (Sanlung) [inline components/layout/Logo.js]: Here the src should point to "../../public/images/Whatodogrey.svg"

File: components/layout/Logo.js [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+            alt="Logos"
+            width="150"
```
Comment (Sanlung) [inline components/layout/Logo.js]: We will need to use responsive CSS (with external stylesheet or styled components) to style the dimensions of the logo later.

**Review by Sanlung — APPROVED** [review]
Great job! I really am all for the look of the logo and will vote to go for it in production. If others are OK, please edit the "src" attribute in the Image component to point to the right source before merging.

File: components/layout/Logo.js (hunk 1 above)
Comment (Asel-K) [reply components/layout/Logo.js:7]: right, will fix it.

Comment (Asel-K) [reply components/layout/Logo.js:7]: I've done that the way you suggested first but ran into an error : 
**Unhandled Runtime Error
Error: Failed to parse src "../public/images/Whatodogrey.svg" on `next/image`, if using relative image it must start with a leading slash "/" or be an absolute URL (http:// or https://)**
and putting "/" solved it.

Comment (Asel-K) [reply components/layout/Logo.js]: There was an error all the time: **Error: Failed to parse src "../../public/images/Logo.svg" on `next/image`, if using relative image it must start with a leading slash "/" or be an absolute URL (http:// or https://)** and putting "/" resolved that. Eventually, I've updated the code and now it meets all the "issues/comments" that you've left  including the width/height of the logo.

Comment (Sanlung) [reply components/layout/Logo.js]: Try "/../../public/images/Whatodogrey.avg then.

**Review by Sanlung — APPROVED** [review]
Sorry if I gave you the impression that the dimensions should be removed. It's OK for now with or without dimensions. It's a go for me.

**Review by dumpidum — APPROVED** [review]
Great job Asel!

**Pull request merged**